book/README.md

```diff
@@ -19,6 +19,21 @@ Documentation on configuring book settings can be found on the [Jupyter Book web
 
 The site is built automatically using these two directories. All of the requirements are specified in `website/requirements.txt`.
 
+#### Locally (Windows)
+
+The following workflow should succeed using a miniconda powershell terminal on Windows 10:
+
+1. conda install git
+2. git clone https://github.com/alan-turing-institute/the-turing-way.git
+3. cd quantecon-mini-example
+4. git checkout windows
+5. conda env create -f environment_win.yml
+6. conda activate wintest
+7. cd mini_book
+8. runjb docs
```

Comment:
Hi!

Unless there's an `environment.yml` file that's not been committed to this PR, we'll have to use `pip` to install the dependencies instead of conda. The workflow looks like this on my machine:

```shell
conda install git
git clone https://github.com/alan-turing-institute/the-turing-way.git
cd ./the-turing-way/
pip install -r requirements
jb build .
```

<img width="992" alt="image" src="https://user-images.githubusercontent.com/8530685/114086945-f57f5080-98bb-11eb-8a95-068a75b4139f.png">